lib/Twig/Extensions/Autoloader.php

```diff
@@ -42,3 +42,5 @@ public static function autoload($class)
         }
     }
 }
+
+class_alias('Twig_Extensions_Autoloader', 'Twig\Extensions\Autoloader', false);
```

Comment:
I would not add an alias here. Instead, I would deprecate this autoloader entirely (we are in the composer time)